deploy/config/php-fpm-auth.conf

```diff
@@ -8,5 +8,9 @@ include fastcgi_params;
 fastcgi_param SCRIPT_NAME $script_name;
 fastcgi_pass fpm;
 
+# Force the request method to get. Otherwise, when verifying POSTs, fpm will be waiting for a non-existing payload.
+# See: https://forum.nginx.org/read.php?2,257968
+fastcgi_param REQUEST_METHOD "GET";
```

Comment:
We need to isolate the setting to the `upload.php` script. Sadly, this approach is too broad.

Reply:
Haha, read the file name 🤦‍♂️ 
`deploy/config/php-fpm-auth.conf`